live-build/config/archives/delphix-secondary-mirror.list.in

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright 2019 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+deb [trusted=yes] @@URL@@ bionic main
```

Comment:
Are we using `trusted=yes` here because we are using aptly to mirror the repository and so we need to re-sign it? I'm assuming we don't need that for the Ubuntu mirror since we copy it "as-is" and do not need to re-sign the repository manifest?

Reply:
@grodr will have to comment on this.. but yes, that's my understanding. I think we can (and probably should, maybe eventually) sign the Aptly repository as well.

Reply:
We are currently using `[trusted=yes]` because I am not resigning yet with aptly, but the plan is to eventually resign, at which point the `trusted` can be removed.